client/client.go

```diff
@@ -105,6 +106,22 @@ type Lister interface {
 	List() ([]string, error)
 }
 
+type Subscription struct {
+	Source string `json:"source" yaml:"source"`
+	Type string `json:"type" yaml:"type"`
+	Broker string `json:"broker" yaml:"broker"`
+}
+
+type FunctionDescription struct {
+	Name          string `json:"name" yaml:"name"`
+	Routes        []string `json:"routes" yaml:"routes"`
```

Comment:
Do we need to assume multiple routes?

Reply:
I think that, at this point, a single route per function is sufficient.  Can you think of a reason for multiples at this time?